code/modules/recycling/disposal.dm

```diff
@@ -127,6 +127,9 @@
 	if(stat & BROKEN || !user || I.flags & ABSTRACT)
 		return
 
+	if(user.a_intent != INTENT_HELP)
+		return
+
 	src.add_fingerprint(user)
 
 	if(istype(I, /obj/item/melee/energy/blade))
```

Comment:
old coders confuse me

Reply:
yeah that part is funny but i guess i better not touch it, maybe it prevents server from exploding?

Reply:
Yeah, nothing needing to be changed here for this PR, its just that old coders confuse me